README.md

```diff
@@ -295,23 +295,28 @@ the language, the arguments to invoke the image with, the expected generated fil
 CONTAINER=namely/gen-grpc-gateway:VVV make test-gwy
 ```
 
+### Auto Dependency Managenement
+
+Dependencies are managed in this repo via [Renovate](https://www.mend.io/free-developer-tools/renovate/).
+Please read more [here](./.github/README.md).
+
 ### Release
 
 Any new gRPC version based release is handled automatically once the relevant [Renovate](https://www.mend.io/free-developer-tools/renovate/) branch is merged to master
 via the CI (Github Action). A patch release should be created manually in Github and the CI workflow will take care of the rest.
 
 #### Contributors
 
-Open a PR and ping one of the Namely employees who have worked on this repo recently.  We will take a look as soon as we can.  
+Open a PR and ping one of the Namely employees who have worked on this repo recently.  We will take a look as soon as we can.\
```

Comment:
Same question about backslash here.

If it's just being used in lieu of extending the line (i.e. the backslash-newline combo is in lieu of a space), note that the file already has longer lines than this one would be.

Reply:
honestly the backslashes were added by a linter I used that matches Codacy. Personally I don't feel strongly about it either way.

Reply:
Huh, ok!